src/site/content/pl/index.njk

```diff
@@ -74,8 +74,20 @@ date: 2018-11-05
     </div>
     <div class="w-grid w-pb--std">
       <div class="w-grid__columns w-grid__columns--gapless w-grid__columns--three" role="list">
-        {% for post in collections.recentBlogPosts %}
-          {% PostCard post %}
+        {% set counter = 0 %}
```

Comment:
Should this look like the en/index.njk example? It seems like you'd add a similar /pl/index.11tydata.js file.

Reply:
Updated